src/routers/middlewares/validation/validateSchema.ts

```diff
@@ -16,3 +16,18 @@ export const validateSchema = (schema: ObjectSchema) => {
         }
     };
 };
+
+export const validateQuerySchema = (schema: ObjectSchema) => {
+    return ({ query }: Request, res: Response, next: NextFunction) => {
+        const { error } = schema.validate(query, {
+            abortEarly: false,
+            allowUnknown: false
+        });
+
+        if (error && error.isJoi) {
```

Comment:
(optional): Looks like you're ignoring non-Joi errors here. All Internal Server Errors should be handled.